src/main/java/org/commcare/formplayer/services/InstallService.java

```diff
@@ -93,10 +93,8 @@ public Pair<FormplayerConfigEngine, Boolean> configureApplication(String referen
             installTimer.record();
             return new Pair<>(engine, newInstall);
         } catch (UnresolvedResourceException e) {
-            log.error("Got exception " + e + " while installing reference " + reference + " at path " + sqliteDB.getDatabaseFileForDebugPurposes());
```

Comment:
The info in these logs is already logged above (reference and database file). Since we are re-throwing the exceptions there's no need to log them here. They will get logged by the main exception handler.